streampark-console/streampark-console-service/src/assembly/script/schema/mysql-schema.sql

```diff
@@ -319,6 +319,24 @@ create table `t_team` (
   unique key `team_name_idx` (`team_name`) using btree
 ) engine = innodb default charset = utf8mb4 collate = utf8mb4_general_ci;
 
+-- ----------------------------
+-- Table of t_variable
+-- ----------------------------
+drop table if exists `t_variable`;
+create table `t_variable` (
+  `id` bigint not null auto_increment,
+  `variable_code` varchar(100) collate utf8mb4_general_ci not null comment 'variable code',
```

Comment:
about `comment` could it be clearer?

Reply:
Ok, I'll describe it in more detail, like it's used as a parameter name or variable placeholder